app/Nova/DuesTransaction.php

```diff
@@ -126,7 +126,7 @@ public function fields(Request $request): array
                 ]
             ),
 
-            MorphMany::make('Payments', 'payment', 'App\Nova\Payment')
+            MorphMany::make('Payments', 'payment', Payment::class)
```

Comment:
Since App\x and App\Nova\x are different, we need to make sure this is referencing the right one

Reply:
each of those files is in the `\App\Nova` namespace so if you say `Payment` that means `\App\Nova\Payment`

Reply:
each of those files is in the `\App\Nova` namespace so if you say `Payment` that means `\App\Nova\Payment`